File: src/main/java/club/deneb/client/features/modules/render/NoHurtCam.java
package club.deneb.client.features.modules.render;


import club.deneb.client.features.Category;
import club.deneb.client.features.Module;

/**
 * @author 086
 */
@Module.Info(name = "NoHurtCam", category = Category.RENDER, description = "Disables the 'hurt' camera effect")
public class NoHurtCam extends Module {

    private static NoHurtCam INSTANCE;

    public NoHurtCam() {
        INSTANCE = this;
    }

    public static boolean shouldDisable() {
        return INSTANCE != null && INSTANCE.isEnabled();
    }

}

File: src/main/java/club/deneb/client/features/modules/render/NoHurtCam.kt
package club.deneb.client.features.modules.render

import club.deneb.client.features.Category
import club.deneb.client.features.Module
import club.deneb.client.features.modules.render.NoHurtCam

@Module.Info(
    name = "NoHurtCam",
    description = "Disables the 'hurt' camera effect",
    category = Category.RENDER
)
class NoHurtCam : Module() {

    init {
        INSTANCE = this
    }

    companion object {
        lateinit var INSTANCE:NoHurtCam
        @JvmStatic
        fun shouldDisable(): Boolean {
            return INSTANCE.isEnabled
        }
    }

}